### ISSUE TITLE
0 value quantity setting should be allowed

### ISSUE THREAD

**Issue by addshore** [maxlath/wikibase-edit#83]
```
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error: missing value
    at Object.new (file:///home/adam/dev/github/addshore/wikibase.world/node_modules/wikibase-edit/lib/error.js:3:17)
    at default (file:///home/adam/dev/github/addshore/wikibase.world/node_modules/wikibase-edit/lib/claim/create.js:7:31)
    at Object.<anonymous> (file:///home/adam/dev/github/addshore/wikibase.world/node_modules/wikibase-edit/lib/bundle_wrapper.js:9:10)
```

- wikibase-edit version: 7.1.2
- Environment: nodejs

I'm guessing this just needs to have a check tightened up somewhere?

Validation that 0 value quantities are fine in a wikibase https://www.wikidata.org/w/index.php?title=Q4115189&diff=prev&oldid=2269514419

**Comment by addshore** [comment]
Looks like it is possible with the slightly more verbose syntax

value: { amount: 0 }

**Comment by maxlath** [comment]
Hello Adam! Fixed by ec9fba0 and published in `v7.1.3`, thanks for reporting!
